src/main/target/common_hardware.c

```diff
@@ -162,6 +162,14 @@
     BUSDEV_REGISTER_I2C(busdev_mag3110,     DEVHW_MAG3110,      MAG3110_I2C_BUS,    0x0E,               NONE,           DEVFLAGS_NONE);
 #endif
 
+#if defined(USE_MAG_LIS3MDL)
+    #if !defined(LIS3MDL_I2C_BUS)
+        #define LIS3MDL_I2C_BUS MAG_I2C_BUS
+    #endif
+    BUSDEV_REGISTER_I2C(busdev_lis3mdl,     DEVHW_LIS3MDL,      LIS3MDL_I2C_BUS,    0x1E,               NONE,           DEVFLAGS_NONE);
+//  BUSDEV_REGISTER_I2C(busdev_lis3mdl,     DEVHW_LIS3MDL,      LIS3MDL_I2C_BUS,    0x1C,               NONE,           DEVFLAGS_NONE);
```

Comment:
If the SDO/SA1 pin of the LIS3MDL is connected to the voltage supply, the address is 0x1E, otherwise, if the SDO/SA1 pin is
connected to ground, the address is 0x1C.